src/lib/rest/rest.cpp

```diff
@@ -1435,15 +1457,25 @@ static int connectionTreat
     ciP->httpStatusCode  = SccContentLengthRequired;
     restReply(ciP, errorMsg);
     alarmMgr.badInput(clientIp, errorMsg);
+    metricsMgr.add(ciP->httpHeaders.tenant, ciP->httpHeaders.servicePath, METRIC_TRANS_IN_ERRORS, 1);
   }
   else if (ciP->answer != "")
   {
     alarmMgr.badInput(clientIp, ciP->answer);
     restReply(ciP, ciP->answer);
+    metricsMgr.add(ciP->httpHeaders.tenant, ciP->httpHeaders.servicePath, METRIC_TRANS_IN_ERRORS, 1);
   }
   else
   {
     serveFunction(ciP);
+
+    //
+    // If a service function sets the httpStatusCode to something above the set of 200s, an error has occurred
+    //
+    if (ciP->httpStatusCode >= SccBadRequest)
```

Comment:
The condition is good but, as discussed at skype, probably all them add METRIC_TRANS_IN_ERROR with 1 could be moved to requestCompleted() function.